src/v2/Apps/Artwork/Components/ArtworkSidebar/index.tsx

```diff
@@ -51,26 +55,19 @@ export const ArtworkSidebar: React.FC<ArtworkSidebarProps> = ({
             <ArtworkSidebarAuctionPartnerInfoFragmentContainer
               artwork={artwork}
             />
-            <ArtworkSidebarAuctionPollingRefetchContainer
-              artwork={artwork}
-              me={me}
-            />
+            {hasEnded ? (
+              <BiddingClosedMessage />
+            ) : (
+              <ArtworkSidebarAuctionPollingRefetchContainer
+                artwork={artwork}
+                me={me}
+              />
+            )}
           </Join>
 
-          {sale &&
-            saleArtwork &&
-            !lotIsClosed(sale, saleArtwork) &&
-            (sale?.cascadingEndTimeInterval ? (
-              <>
-                <Spacer mt={2} />
-                <LotTimerFragmentContainer saleArtwork={saleArtwork} />
-              </>
-            ) : (
-              <>
-                <Spacer mt={2} />
-                <AuctionTimerFragmentContainer sale={sale} />
-              </>
-            ))}
+          {!hasEnded && (
+            <ArtworkSidebarAuctionTimerFragmentContainer artwork={artwork} />
+          )}
```

Comment:
This might be more clear if you split this up into what shows when the lot has ended vs. when the lot hasn't ended (or isn't cascade).

```tsx
{hasEnded ? (
              <BiddingClosedMessage />
            ) : (
             <>
                <ArtworkSidebarAuctionPollingRefetchContainer
                  artwork={artwork}
                  me={me}
                />
               <ArtworkSidebarAuctionTimerFragmentContainer 
                artwork={artwork} />
             </>
            )}
          </Join>

Reply:
Did I understand this correctly?

Reply:
Yea, what you posted is the intent, essentially.

The reason why it's split up (and so there's a check on `hasEnded` twice, instead of once), is b/c  the auction timer actually lives _outside_ of the `Join` that the polling container is a part of. I did see a spacing issue when I combined them as you suggested. (but of course it is likely possible to address that)

Reply:
Oh got it, I don't feel strongly at all, so feel free to keep it as is so you don't have to worry about the spacing.